Procfile.dev

```diff
@@ -0,0 +1,3 @@
+web: bin/rails server --binding 0.0.0.0 --port 3000
+css: yarn build:css --watch
```

Comment:
Fine for me to have a separate service for `css` and `js`. Would it be possible to merge the two?